core-primitives/ocall-api/src/lib.rs

```diff
@@ -115,7 +115,8 @@ pub trait EnclaveSidechainOCallApi: Clone + Send + Sync {
 
 	fn fetch_sidechain_blocks_from_peer<SignedSidechainBlock: Decode>(
 		&self,
-		last_known_block_hash: BlockHash,
+		last_imported_block_hash: BlockHash,
+		maybe_until_block_hash: Option<BlockHash>,
```

Comment:
Additional, optional, parameter to specify an 'until' block hash when fetching sidechain blocks from peer